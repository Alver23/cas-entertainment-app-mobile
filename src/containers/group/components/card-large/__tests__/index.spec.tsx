// Dependencies
import React from 'react';
import { Text } from 'react-native';
import { shallow, ShallowWrapper } from 'enzyme';

// Components
import CardLarge from '../index';

describe('<CardLarge />', () => {
  let component: ShallowWrapper;

  const props = {};

  beforeEach(() => {
    component = shallow(<CardLarge {...props} />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
    expect(component.find(Text).exists()).toBeTruthy();
  });
});
