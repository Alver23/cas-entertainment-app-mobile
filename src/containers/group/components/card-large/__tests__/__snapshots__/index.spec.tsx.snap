// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardLarge /> should save a snapshot of the component 1`] = `
<Fragment>
  <Text>
    CardLarge
  </Text>
</Fragment>
`;
